Tasks/UsePythonVersionV0/usepythonversion.ts

```diff
@@ -37,7 +37,7 @@ function binDir(installDir: string, platform: Platform): string {
     }
 }
 
-function pypyNotFoundError(versionSpec: '2' | '3.6') {
+function pypyNotFoundError(versionSpec: string) {
```

Comment:
The change from `pypy3` -> `pypy3.6` being NECESSARY is what I'd consider the breaking change in this PR. It probably should be bumped to version `@1`.